### ISSUE TITLE
It doesn’t support for iOS Apps.

### ISSUE THREAD

**Issue by bharath3648** [nitish24p/react-native-upi#9]
It doesn’t support for iOS.

**Comment by nitish24p** [comment]
Yes because i've never written code in objective C or swift or IOS for that reason.. If you would want to contribute, would be happy to merge a PR

**Comment by bharath3648** [comment]
Thanks for the update @nitish24p 

But, something went wrong with react-native-image-picker after installing react-native-upi-payment.
Crashing after selecting image(react-native-image-picker: "^1.1.0"/"^0.28.0")

-> If we are using the above package, app getting crashes after selecting/capturing an image without any error, if I install UPI payment package ("react-native-upi-payment": "^1.0.2")

-> Everything is like a document provided by related packages.

**Comment by vishwanath-ogit** [comment]
> Thanks for the update @nitish24p
> 
> But, something went wrong with react-native-image-picker after installing react-native-upi-payment.
> Crashing after selecting image(react-native-image-picker: "^1.1.0"/"^0.28.0")
> 
> -> If we are using the above package, app getting crashes after selecting/capturing an image without any error, if I install UPI payment package ("react-native-upi-payment": "^1.0.2")
> 
> -> Everything is like a document provided by related packages.

Same issue I am facing

**Comment by UnSensei** [comment]
@bharath3648 @vishwanath-ogit can u check if its still crashes when u remove dont unse crop?
